src/neo-express/Node/NodeUtility.cs

```diff
@@ -172,23 +173,42 @@ void SubmitTransaction(Func<Snapshot, WalletAccount, Transaction?> factory)
                 Blockchain.GoverningToken.Hash));
         }
 
-        public static Task RunAsync(Store store, ExpressConsensusNode node, TextWriter writer, uint preloadGas, CancellationToken cancellationToken)
+        public static void Preload(uint preloadGasAmount, Store store, ExpressConsensusNode node, TextWriter writer, CancellationToken cancellationToken)
         {
-            static bool PreloadValid()
-            {
-                using var snapshot = Blockchain.Singleton.GetSnapshot();
-                var lastBlock = Blockchain.Singleton.GetBlock(snapshot.CurrentBlockHash);
-                var validators = snapshot.GetValidators();
-                return lastBlock.Index == 0 && validators.Length == 1;
-            }
+            Debug.Assert(preloadGasAmount > 0);
+
+            var wallet = DevWallet.FromExpressWallet(node.Wallet);
+            using var system = new NeoSystem(store);
+
+            system.StartNode(node.TcpPort, node.WebSocketPort);
 
-            static uint CalculateGas(uint preloadGas)
+            var generationAmount = Blockchain.GenerationAmount[0];
+            var gas = preloadGasAmount / generationAmount;
+            var preloadCount = preloadGasAmount % generationAmount == 0 ? gas : gas + 1;
+            writer.WriteLine($"Creating {preloadCount} empty blocks to preload {preloadGasAmount} GAS");
+            Random random = new Random();
+            for (int i = 1; i <= preloadCount; i++)
             {
-                var generationAmount = Blockchain.GenerationAmount[0];
-                var gas = preloadGas / generationAmount;
-                return preloadGas % generationAmount == 0 ? gas : gas + 1;
+                if (i % 100 == 0)
+                {
+                    writer.WriteLine($"  Creating Block {i}");
+                }
+                var block = CreatePreloadBlock(wallet, random);
+                var relayResult = system.Blockchain.Ask<RelayResultReason>(block).Result;
```

Comment:
Nice job, @devhawk.
Good workaround.

Reply:
If we create all the preload blocks on a single node, wouldn't those blocks automatically sync to the other nodes automatically? 

Filed #46 to track multi-node support

Reply:
Yes, other nodes will sync automatically, @devhawk.

Reply:
THen it's probably just a matter of updating the signing code. Will do that as part of #46